test_scripts/Defects/8_1/980_SDL_responds_INVALID_DATA_in_case_soft_button_has_Type_Image_or_Both_and_Text_has_specific_string.lua

```diff
@@ -0,0 +1,89 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/980
+---------------------------------------------------------------------------------------------------
+-- Description: Check that SDL responds INVALID_DATA in case soft button has Type(Image or Both)
+-- and Text is whitespace or \t or \n or empty
+--
+-- Precondition:
+-- 1) AlertManeuver is allowed by policy
+-- 2) SDL, HMI, Mobile session started
+-- 3) App is registered and activated
+-- In case:
+-- 1) App sends AlertManeuver with soft button (type = Image or Both and Text = whitespace or \t or \n or empty)
+-- SDL does:
+-- 1) Respond with AlertManeuver(success = false, resultCode = "INVALID_DATA") to app
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require("user_modules/sequences/actions")
+local utils = require('user_modules/utils')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local requestParams = {
+  type_BOTH_Text_whitespace = {
+    ttsChunks = { { text = "FirstAlert", type = "TEXT" } },
+    softButtons = { { type = "BOTH", text = "        ", softButtonID = 821, } }
+  },
+  type_BOTH_Text_HorizontalTab = {
+    ttsChunks = { { text = "FirstAlert", type = "TEXT" } },
+    softButtons = { { type = "BOTH", text = "Close\t", softButtonID = 821, } }
```

Comment:
Please note that according to req https://adc.luxoft.com/jira/browse/APPLINK-20836?src=confmacro
In case mobile app sends any relevant-RPC with <SoftButtons> with "Type" = BOTH
and at least one of the parameters ('text' and 'image') wrong or not defined,
SDL must:
respond with _'INVALID_DATA, success:false" resultCode to mobile app
SDL must NOT:
transfer this RPC to HMI.
Please add image parameter for softButtons to check the fix